KubeArmor/BPF/tests/Makefile

```diff
@@ -0,0 +1,104 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This makefile is duplicated. Can we do better here? May be we can leverage makefile's `include` construct.

Reply:
Done